packages/xo-web/src/common/xo/index.js

```diff
@@ -1251,6 +1251,10 @@ export const migrateVm = async (vm, host) => {
     return error(_('migrateVmNoTargetHost'), _('migrateVmNoTargetHostMessage'))
   }
 
+  if (params.targetSrs === undefined) {
```

Comment:
The SR select will be hidden in case of migrating a VM, contained on a shared SR, without selecting a network. 

 IMHO, we should always display the select if we assume that the target SR is required, .

Reply:
> The SR select will be hidden in case of migrating a VM, contained on a shared SR, without selecting a network.

 You're right, I'll take that into account.

> IMHO, we should always display the select if we assume that the target SR is required.

  Intra-pool: The SR is only required when selecting a network.